staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation_test.go

```diff
@@ -90,7 +90,7 @@ func TestValidateCustomResourceDefinition(t *testing.T) {
 		{
 			name: "invalid types allowed via v1beta1",
 			resource: &apiextensions.CustomResourceDefinition{
-				ObjectMeta: metav1.ObjectMeta{Name: "plural.group.com"},
+				ObjectMeta: metav1.ObjectMeta{Name: "plural.group.com", Tenant: testTenant},
```

Comment:
Perhaps we need to keep default tenant (system) and add tests for testTenant

Reply:
double checked all tests in this test case using Tenant: testTenant from arktos PR: Change crd scope #224